.github/workflows/ci-base-tests-linux.yml

```diff
@@ -49,5 +62,36 @@ jobs:
             --ignore=./smarts/core/tests/test_smarts_memory_growth.py \
             --ignore=./smarts/core/tests/test_env_frame_rate.py \
             --ignore=./smarts/env/tests/test_benchmark.py \
-            --ignore=./examples/tests/test_learning.py \
             -k 'not test_long_determinism'
+
+  examples-rl:
```

Comment:
How long do these two tests take to run?

Reply:
+ Currently the `examples-rl` test takes [~20 minutes](https://github.com/huawei-noah/SMARTS/actions/runs/5248833298)
+ By reducing the number of routes generated in the scenarios, the `examples-rl` test time may be reduced to [~12 minutes](https://github.com/huawei-noah/SMARTS/actions/runs/5249494423)
+ Previously the `base-tests` test took [~53 minutes](https://github.com/huawei-noah/SMARTS/actions/runs/5248804421) to run when it attempted to build all scenarios.
+ By only building the necessary scenarios, the `base-tests` now takes [~6 minutes](https://github.com/huawei-noah/SMARTS/actions/runs/5248833298).
+ For a quick benefit, we may make another PR consisting of only building the necessary scenarios which will hasten the `base-tests`, while we figure how to best do CI for `examples-rl` test.